src/Appwrite/Utopia/Response/Model/UsageUsers.php

```diff
@@ -5,72 +5,61 @@
 use Appwrite\Utopia\Response;
 use Appwrite\Utopia\Response\Model;
 
+/**
+ * Class UsageUsers
+ *
+ * Model class representing a usage statistics for users and sessions.
+ */
 class UsageUsers extends Model
 {
+    /**
+     * Constant representing the type string.
+     *
+     * @var string
+     */
+    const TYPE_STRING = 'string';
+    const TYPE_METRIC = 'metric';
+
     public function __construct()
     {
+        $commonProperties = [
+            'type' => self::TYPE_METRIC,
+            'description' => '',
+            'default' => [],
+            'example' => [],
+            'array' => true
+        ];
+
         $this
             ->addRule('range', [
                 'type' => self::TYPE_STRING,
                 'description' => 'The time range of the usage stats.',
                 'default' => '',
                 'example' => '30d',
             ])
-            ->addRule('usersCount', [
-                'type' => Response::MODEL_METRIC,
+            ->addRule('usersCount', $commonProperties + [
                 'description' => 'Aggregated stats for total number of users.',
-                'default' => [],
-                'example' => [],
-                'array' => true
             ])
```

Comment:
I think this works differently than expected...According to the [docs](https://www.php.net/manual/en/language.operators.array.php), the keys in the right array do not replace the matching ones in the left array.

This type of nuance with how the operator works suggest it might be simpler and clearer to leave it the way it is.